data/meson.build

```diff
@@ -26,13 +29,14 @@ if build_standalone
     install_dir: join_paths(datadir, 'fwupd', 'quirks.d'))
 endif
 
-install_data(['org.freedesktop.fwupd.metainfo.xml'],
-  install_dir: join_paths(datadir, 'metainfo')
-)
-
-install_data(['org.freedesktop.fwupd.svg'],
-  install_dir : join_paths(datadir, 'icons', 'hicolor', 'scalable', 'apps')
-)
+if get_option('metainfo')
```

Comment:
we can probably exclude this from windows build too now that this option exists

Reply:
Yes, and FreeBSD perhaps too?

Reply:
No, FreeBSD makes sense to keep. Ignore me.

Reply:
Done